cmd/metrics-router.go

```diff
@@ -24,7 +24,9 @@ import (
 )
 
 const (
-	prometheusMetricsPath = "/prometheus/metrics"
+	prometheusMetricsPath          = "/prometheus/metrics"
+	prometheusMetricsV2ClusterPath = "/prometheus/metrics/cluster"
+	prometheusMetricsV2NodePath    = "/prometheus/metrics/node"
```

Comment:
```suggestion
	prometheusMetricsV2ClusterPath = "/v2/metrics/cluster"
	prometheusMetricsV2NodePath    = "/v2/metrics/node"
```

how about we make this more simple?

Reply:
that is cleaner.